src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs

```diff
@@ -192,20 +194,50 @@ public static IResourceBuilder<RedisResource> WithRedisInsight(this IResourceBui
             return builder;
         }
 
-        static async Task ImportRedisDatabases(ILogger resourceLogger, IEnumerable<RedisResource> redisInstances, HttpClient client, CancellationToken ct)
+        static async Task ImportRedisDatabases(ILogger resourceLogger, IEnumerable<RedisResource> redisInstances, HttpClient client, CancellationToken cancellationToken)
```

Comment:
Simpler to just clear it everytime instead?

Reply:
I was thinking that if someone had manually configured Redis connection to another container then they might prefer if we didn't delete it.

I had contemplated not removing them and just not importing if the name matched, but that wouldn't work with password change occurrences.